test/bandit/http1/request_test.exs

```diff
@@ -351,7 +351,7 @@ defmodule HTTP1RequestTest do
       response =
         Req.get!(context.req,
           url: "/expect_headers/a//b/c?abc=def",
-          headers: [{"X-Fruit", "banana"}]
+          headers: [{"x-fruit", "banana"}]
```

Comment:
Req automatically downcases header names (and so does Mint. which is doing the same work twice and yeah I'd like to do something about it.) so I thought being more explicit here would be helpful.